GirafIntegrationTest/tests/test_account_controller.py

```diff
@@ -0,0 +1,355 @@
+from requests import get, post, put, delete
+import time
+from testlib import order, BASE_URL, auth, GIRAFTestCase
+
+graatand_token = ''
+graatand_id = ''
+gunnar_token = ''
+gunnar_reset_token = ''
+gunnar_username = ''
+gunnar_id = ''
+grundenberger_token = ''
+grundenberger_id = ''
+grundenberger_username = ''
+tobias_token = ''
+
+
+class TestAccountController(GIRAFTestCase):
+    """
+    Testing API requests on Account endpoints
+    """
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        """
+        Setup necessary data when class is loaded
+        """
+        super(TestAccountController, cls).setUpClass()
+        print(f'file:/{__file__}\n')
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        """
+        Remove or resolve necessary data and states after class tests are done
+        """
+        super(TestAccountController, cls).tearDownClass()
+
+    def setUp(self) -> None:
+        """
+        Setup necessary data and states before each test
+        """
+        pass
+
+    def tearDown(self) -> None:
+        """
+        Remove or resolve necessary data and states after each test
+        """
+        pass
+
+    @order
+    def test_account_login_as_graatand(self):
+        """
+        Testing logging in as Graatand
+
+        Endpoint: POST:/v1/Account/login
+        """
+        global graatand_token
+        data = {'username': 'Graatand', 'password': 'password'}
+        response = post(f'{BASE_URL}v1/Account/login', json=data).json()
+        self.assertTrue(response['success'])
+        self.assertEqual(response['errorKey'], 'NoError')
+        self.assertIsNotNone(response['data'])
+        graatand_token = response['data']
+
+    @order
+    def test_account_get_graatand_id(self):
+        """
+        Testing getting Graatand's id
+
+        Endpoint: GET:/v1/User
+        """
+        global graatand_id
+        response = get(f'{BASE_URL}v1/User', headers=auth(graatand_token)).json()
+        self.assertTrue(response['success'])
+        self.assertEqual(response['errorKey'], 'NoError')
+        self.assertIsNotNone(response['data']['id'])
+        graatand_id = response['data']['id']
+
+    @order
+    def test_account_register_grundenberger(self):
+        """
+        Testing registering Grundenberger
+
+        Endpoint: POST:/v1/Account/register
+        """
+        global grundenberger_username
+        grundenberger_username = f'Grundenberger{time.time()}'
+        data = {'username': grundenberger_username, 'password': 'password', 'role': 'Citizen', 'departmentId': 1}
+        response = post(f'{BASE_URL}v1/Account/register', json=data, headers=auth(graatand_token)).json()
+        self.assertTrue(response['success'])
+        self.assertEqual(response['errorKey'], 'NoError')
+
+    @order
+    def test_account_login_as_grundenberger(self):
```

Comment:
```suggestion
    def test_can_login_as_grundenberger(self):
```

Reply:
The way I named the functions is `test_domain_...`. The reason for this is that `unittest` puts all tests in a list rather than a dict - i.e. if you log in as the same user in several files you would likely give the test the same name, so said method would only be added once. Maybe `test_account_can_login_as_grundenberger` instead?

Otherwise the `TestSuite` class would have to be modified to use a dict instead.

Reply:
Alright. But isn't it a bit confusing for people that they are required to write `test_domain_...` when running the tests?

But well, if it says so in the wiki, maybe it is not that big of a problem.

Reply:
My assumption, and I presume the original authors as well, is that all tests are run at once through `test.py`, given that they are integration tests. Perhaps an argparse gimmick to run only a single file could be a reasonable solution, if you don't want to run all tests at once? 

Both the above and what I mentioned with data structure in the comment above are likely out of scope for this PR though.